Sources/Shared/Request+Root.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Request+Root.swift
+//  VimeoNetworking
+//
+//  Copyright © 2019 Vimeo. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import Foundation
+
+/// `Request` that returns an array of `UserItem` objects.
+public typealias RootRequest = Request<[UserItem]>
+
+public extension Request {
+    
+    /// Returns a new request to fetch an array of user items.
+    ///
+    /// - Parameter userURI: The user's URI.
+    /// - Returns: Returns a new `Request` for an array of user items.
+    static func rootRequest(for userURI: String) -> Request {
```

Comment:
```suggestion
    static func folderRootRequest(for userURI: String) -> Request<[UserItem]> {
```

Reply:
There should also be another method:
```swift
static func folderRoorRequestForMe() -> Request<[UserItem]>
```

This method should simply use `/me/folders/root` and should not use a supplied `userURI`.

Reply:
Per our test this is not available at this time.